src/oneD/StFlow.cpp

```diff
@@ -484,6 +487,14 @@ void StFlow::updateTransport(doublereal* x, size_t j0, size_t j1)
                 m_trans->getThermalDiffCoeffs(m_dthermal.ptrColumn(0) + j*m_nsp);
             }
         }
+    } else if (m_do_unitary_lewis) {// unitary lewis number
```

Comment:
If the unity Le class overrides the correct function, then there does not need to be a special case here and it can be treated the same way as the mixture averaged model.